doc/openapi/v3/device-onvif-camera.yaml

```diff
@@ -8242,20 +8371,49 @@ components:
             $ref: '#/components/schemas/analytics_ConfigOptions'
           type: array
       type: object
+    analytics_GetRules:
+      properties:
+        ConfigurationToken:
+          description: Reference to an existing VideoAnalyticsConfiguration.
+          maxLength: 64
+          type: string
+      required:
+      - ConfigurationToken
+      type: object
     analytics_GetRulesResponse:
       properties:
         Rule:
           items:
             $ref: '#/components/schemas/onvif_Config'
           type: array
       type: object
+    analytics_GetSupportedAnalyticsModules:
+      properties:
+        ConfigurationToken:
+          description: Reference to an existing VideoAnalyticsConfiguration.
+          maxLength: 64
+          type: string
+      required:
+      - ConfigurationToken
+      type: object
     analytics_GetSupportedAnalyticsModulesResponse:
       properties:
         SupportedAnalyticsModules:
           $ref: '#/components/schemas/onvif_SupportedAnalyticsModules'
       required:
       - SupportedAnalyticsModules
       type: object
+    analytics_GetSupportedRules:
+      properties:
+        ConfigurationToken:
+          description: "References an existing Video Analytics configuration. The\
+            \ list of available tokens can be obtained\n\t\t\t\t\t\t\t\t\tvia the\
```

Comment:
Looks like there are some extra tabs may be its because its auto generated. The extras should be removed?

Reply:
I will take a look at if this is a bug in the translation, or if the original had them as well. Either way we can probably replace them, however I would be careful about that, as there may be some places they use the tabs to format for other reasons than block-indenting (which is what they are using it for here).

Reply:
@presatish I have addressed this. They are now multi-line raw strings (and have had the tabs removed)